lib/api/cln/cln_client.dart

```diff
@@ -109,4 +112,29 @@ class CLNApi extends AppApi {
         onDecode: (jsonResponse) =>
             AppListChannels.fromJSON(jsonResponse as Map<String, dynamic>));
   }
+
+  @override
+  Future<AppPayInvoice> payInvoice({required String invoice, int? msat}) async {
+    dynamic params;
+    switch (mode) {
+      case ClientMode.grpc:
+        if (msat == -1) {
+          params = CLNPayRequest(grpcRequest: PayRequest(bolt11: invoice));
+        } else {
```

Comment:
```suggestion
        if (msat == -1) {
          params = CLNPayRequest(grpcRequest: PayRequest(bolt11: invoice));
        } else {
```

Why this -1 here? why not using the null value directly